src/bp-activity/bp-activity-template.php

```diff
@@ -2762,6 +2762,11 @@ function bp_get_activity_css_class() {
 		$class .= ' bb-pinned';
 	}
 
+	$activity_mute_notification_meta = bp_activity_get_meta( bp_get_activity_id(), 'muted_notification_users' );
```

Comment:
@kishanbuddyboss Use this function `bb_user_has_mute_notification`.

Reply:
Fixed it